### ISSUE TITLE
Diff Monitor and Chart Monitor not working?

### ISSUE THREAD

**Issue by mondaychen** [zalmoxisus/redux-devtools-extension#92]
With the extension installed I tried it with [the todomvc example](http://zalmoxisus.github.io/redux-devtools-extension/examples/todomvc/). The Log Monitor and Slider Monitor both work very well. However the Chart Monitor is all blank and Diff Monitor does not show any diff.

Not sure if it only happens to me.

My environment:

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.112 Safari/537.36
```

**Comment by truckingsim** [comment]
Also having the exact same issues.

```
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.75 Safari/537.36
```

package.json important bits, anything relating to redux (just in case):
```js
{
  "dependencies": {
    "react-redux": "^4.4.0",
    "react-router-redux": "^4.0.2"
    "redux": "^3.4.0",
    "redux-form": "^4.2.0",
    "redux-thunk": "^1.0.3"
  }
}
```

**Comment by zalmoxisus** [comment]
@mondaychen, @truckingsim, thanks for the report! Downgrading versions of those monitors fixes the issue. I will investigate what changes there caused the problem. 

BTW are there any advantages of supporting `Diff Monitor` since `Inspector Monitor` also shows diffs?

**Comment by zalmoxisus** [comment]
Fixed in `v1.0.0.21`. Please give it a try and feel free to reopen the issue if the problem still appears.

**Comment by truckingsim** [comment]
@zalmoxisus Looks fixed.  Thanks!
